test/react_client_test.dart

```diff
@@ -138,24 +138,25 @@ main() {
         originalHandlers = {};
         props = {};
 
-        for (final key in eventPropKeyToEventFactory.keys) {
+        for (final key in knownEventKeys) {
           props[key] = originalHandlers[key] = createHandler(key);
         }
       });
 
       test('for a DOM element', () {
         var component = react.div(props);
         var jsProps = unconvertJsProps(component);
-        for (final key in eventPropKeyToEventFactory.keys) {
+        for (final key in knownEventKeys) {
           expect(jsProps[key], isNotNull, reason: 'JS event handler prop should not be null');
-          expect(jsProps[key], same(originalHandlers[key]), reason: 'JS event handler prop was not unconverted');
+          expect(jsProps[key], anyOf(same(originalHandlers[key]), same(allowInterop(originalHandlers[key]))),
+              reason: 'JS event handler prop was not unconverted');
```

Comment:
I think this message is outdated now
```suggestion
              reason: 'JS event handler be the original or original wrapped in allowInterop');
```